cmd/cem.go

```diff
@@ -0,0 +1,395 @@
+}
+
+var cemGetRecommendationsCmd = &cobra.Command{
+	Use:   "get-recommendations",
+	Short: "Get Entity Recommendations",
+	Long: `Retrieve recommendations for an entity
+
+	Example Usage:
+	$ cybr cem get-recommendations -p PLATFORM -a ACCOUNT_ID -e ENTITY_ID`,
+	Run: func(cmd *cobra.Command, args []string) {
+		token, err := GetToken()
+		if err != nil {
+			log.Fatalf("Failed to retrieve token file at %s. %s\n", CemSessionTokenPath, err)
+		}
+
+		cemQuery := &cem.EntityQuery{
+			Platform:  CemPlatform,
+			AccountID: CemAccountID,
+			EntityID:  CemEntityID,
+		}
+
+		cemResult, err := cem.GetEntityRecommendations(token, cemQuery)
+		if err != nil {
+			log.Fatalln(err)
+			return
+		}
+
+		prettyprint.PrintJSON(cemResult)
+
+	},
+}
+
+var cemGetEntityDetailCmd = &cobra.Command{
+	Use:   "get-entity-detail",
+	Short: "Get Entity Details",
+	Long: `Retrieve the details of a specific entity within a platform and workspace
+
+	Example Usage:
+	$ cybr cem get-entity-detail -p PLATFORM -a ACCOUNT_ID -e ENTITY_ID`,
+	Run: func(cmd *cobra.Command, args []string) {
+		token, err := GetToken()
+		if err != nil {
+			log.Fatalf("Failed to retrieve token file at %s. %s\n", CemSessionTokenPath, err)
+		}
+
+		cemQuery := &cem.EntityQuery{
+			Platform:  CemPlatform,
+			AccountID: CemAccountID,
+			EntityID:  CemEntityID,
+		}
+
+		cemResult, err := cem.GetEntityDetail(token, cemQuery)
+		if err != nil {
+			log.Fatalln(err)
+			return
+		}
+
+		prettyprint.PrintJSON(cemResult)
+	},
+}
+
+var cemGetEntitiesCmd = &cobra.Command{
+	Use:   "get-entities",
+	Short: "Get Entities",
+	Long: `Search for entities on any platform and retrieve entity details.
+	
+	Example Usage:
+	$ cybr cem get-entities -p PLATFORM -a ACCOUNT_ID`,
+	Run: func(cmd *cobra.Command, args []string) {
+		token, err := GetToken()
+		if err != nil {
+			log.Fatalf("Failed to retrieve token file at %s. %s\n", CemSessionTokenPath, err)
+		}
+
+		if CemNonFullAdmin && CemFullAdmin {
+			log.Fatal("Cannot set both --non-full-admin and --full-admin at the same time")
+		}
+
+		if CemNonShadowAdmin && CemShadowAdmin {
+			log.Fatal("Cannot set both --non-shadow-admin and --shadow-admin at the same time")
+		}
+
+		inputFullAdmin := ""
+		inputShadowAdmin := ""
+
+		if CemFullAdmin {
+			inputFullAdmin = "true"
+		}
+		if CemNonFullAdmin {
+			inputFullAdmin = "false"
+		}
+		if CemShadowAdmin {
+			inputShadowAdmin = "true"
+		}
+		if CemNonShadowAdmin {
+			inputShadowAdmin = "false"
+		}
+
+		cemQuery := &cem.GetEntitiesQuery{
+			Platform:    CemPlatform,
+			AccountId:   CemAccountID,
+			FullAdmin:   inputFullAdmin,
+			ShadowAdmin: inputShadowAdmin,
+			NextToken:   CemNextToken,
+		}
+
+		cemResult, err := cem.GetEntities(token, cemQuery)
+		if err != nil {
+			log.Fatalln(err)
+			return
+		}
+
+		prettyprint.PrintJSON(cemResult)
+	},
+}
+
+func init() {
+
+	// Login
+	cemLoginCmd.Flags().StringVarP(&CemOrganization, "organization", "o", "", "Username to logon PAS REST API using")
```

Comment:
I believe that this description is incorrect.

`Organization ID for CEM` or something?